packages/ui-components/src/Icon/README.md

```diff
@@ -0,0 +1,50 @@
+[back to components](../README.md)
+
+# Icon
+
+An icon is a graphic element to quickly communicate a concept without or in partnership with text content. An Icon is an SVG element intended to be used individualy to render an SVG graphic or consumed by other components to render icons.
+
+```javascript
+import React from "react";
+import { Badge } from "@cockroachlabs/ui-components";
+
+export SomeExampleElement = () => (
+  <Icon iconName="CheckCircle" />
+  <Icon iconName="ErrorCircle" tint="red" />
+  <Icon iconName="Caution" size="small" />
+  <Icon iconName="World" tint="blue" size="large" />
+);
+```
+
+## Properties
+
+These are the properties that can be provided to the `Icon` component.
+
+### `iconName: string`
+
+The `iconName` prop should be a valid icon name from the [icons package](https://github.com/cockroachdb/ui/tree/master/packages/icons#icons). If an invalid iconName is supplied, the component will return `null`.
+
+### `size?: Enum("large", "medium", "small", "tiny")`
```

Comment:
I'd be more comfortable if prop options were documented by generated stories rather than us updating them every time we change the component.

Reply:
Are you suggesting I delete this README? Or that we use stories as the only component documentation?

Reply:
I think everything below "Properties" should be removed since it's not sustainable to maintain. Just having the properties documented in generated stories and by the code should cover all of our use cases. I don't have strong feelings on where the description and example should live but I imagine we would want to have a those in a single location as well. What do you think?

Reply:
I think since this is causing confusion it should be removed. I like the idea of having documentation for these components in the same repo as the code, but agree that the maintenance is a more important issue. Rachel and Anne are exploring plugins for storybook that I think would be the best place for all documentation including props, usage, and code examples. After I wrap this component, I'll circle back to cleaning up the READMEs and migrating example usage into storybook.